package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -0,0 +1,420 @@
+                        +-- units <str>
+                \-- (velocity)
+                    \-- [step] <int>, gives frame
+                    \-- [time] <float>, gives time
+                        +-- units <str>
+                    \-- [value] <float>, gives numpy arrary of velocities
+                                         with shape (n_atoms, 3)
+                        +-- units <str>
+                \-- (force)
+                    \-- [step] <int>, gives frame
+                    \-- [time] <float>, gives time
+                        +-- units <str>
+                    \-- [value] <float>, gives numpy arrary of forces
+                                         with shape (n_atoms, 3)
+                        +-- units <str>
+                \-- (data)
+                    \-- (dt)
+                        \-- [step] <int>, gives frame number
+                        \-- [value] <float>, gives dt
+                    \-- (lambda)
+                        \-- [step] <int>, gives frame
+                        \-- [value] <float>, gives lambda
+                    \-- (step)
+                        \-- [step] <int>, gives frame
+                        \-- [value] <int>, gives step
+
+
+    .. note::
+        The reader does not currently read mass data.
+
+    """
+
+    format = 'H5MD'
+    # units are added from h5md file
+    units = {'time': None,
+             'length': None,
+             'velocity': None,
+             'force': None}
+    # Translate H5MD units (https://nongnu.org/h5md/modules/units.html)
+    # to MDAnalysis units.
+    _unit_translation = {
+        'time': {
+            'ps': 'ps',
+            'fs': 'fs',
+            'ns': 'ns',
+            'second': 'second',
+            'sec': 'sec',
+            's': 's',
+            'AKMA': 'AKMA',
+            },
+        'length': {
+            'Angstrom': 'Angstrom',
+            'angstrom': 'Angstrom',
+            'A': 'Angstrom',
+            'nm': 'nm',
+            'pm': 'pm',
+            'fm': 'fm',
+            },
+        'velocity': {
+            'Angstrom ps-1': 'Angstrom/ps',
+            'A ps-1': 'Angstrom/ps',
+            'Angstrom fs-1': 'Angstrom/fs',
+            'A fs-1': 'Angstrom/fs',
+            'Angstrom AKMA-1': 'Angstrom/AKMA',
+            'A AKMA-1': 'Angstrom/AKMA',
+            'nm ps-1': 'nm/ps',
+            'nm ns-1': 'nm/ns',
+            'pm ps-1': 'pm/ps',
+            'm s-1': 'm/s'
+            },
+        'force':  {
+            'kJ mol-1 Angstrom-1': 'kJ/(mol*Angstrom)',
+            'kJ mol-1 nm-1': 'kJ/(mol*nm)',
+            'Newton': 'Newton',
+            'N': 'N',
+            'J m-1': 'J/m',
+            'kcal mol-1 Angstrom-1': 'kcal/(mol*Angstrom)',
+            'kcal mol-1 A-1': 'kcal/(mol*Angstrom)'
+            }
+    }
+    _Timestep = Timestep
+
+    def __init__(self, filename, convert_units=True, **kwargs):
+        """
+        Parameters
+        ----------
+        filename : str or h5py.File
```

Comment:
```suggestion
        filename : str or :class:`h5py.File`
```

Reply:
If you add to https://github.com/MDAnalysis/mdanalysis/blob/2e6b9608bfdbbf02133d5d1f0cd29db0cd43473b/package/doc/sphinx/source/conf.py#L340 the line
```
'https://docs.h5py.org/en/stable/': None,
```
then this will automatically link to the h5py docs.